tests/unittests/test_error_handling.py

```diff
@@ -2,8 +2,11 @@
 from unittest import mock
```

Comment:
Please add comments for each test, like what it is testing.

Reply:
Added